fpmsyncd/routesync.h

```diff
@@ -77,6 +99,12 @@ class RouteSync : public NetMsg
     ProducerStateTable  m_vnet_tunnelTable; 
     struct nl_cache    *m_link_cache;
     struct nl_sock     *m_nl_sock;
+#ifdef HAVE_NEXTHOP_GROUP
+    /* nexthop group table */
+    ProducerStateTable  m_nexthop_groupTable;
+    map<uint32_t,NextHopGroup> m_nh_groups;
+    map<string,NextHopGroupRoute> m_nh_routes;
```

Comment:
The fpmsyncd should NOT cache the routes, it will consume too much memory in large scale routes scenario.
For one route entry, it already exists in orchagent, sai meta, syncd(sai/sdk). We cannot afford another copying :(

Reply:
Removed `map<string,NextHopGroupRoute> m_nh_routes;` based on the discussion in the Routing WG.
Please kindly check if this has resolved your concern.